Adds CarpentryCon 2020's guide for proposal submission post (#597)
carpentries/carpentries.org: content/posts/2026/01/new-partnership-program-is-launching-in-april.md

---
layout: page
authors: ["Erin Becker", "Oscar Masinyana", "Angelique Trusler"]
teaser: "We have redesigned The Carpentries Membership program into a Partnership Program that can be customised to fit varying institutional needs."
title: "The Carpentries’ New Partnership Program is Launching in April!"
date: 2026-01-29
time: "09:00:00"
tags: ["Membership", "Community"]
---

## Introducing the new Partnership Program

> With much valued input from our community, we have been working over the past few years on ways to improve our Membership program in order to meet the needs of our community’s partner organisations and institutions. After considering and engaging with feedback from our community surveys from [September 2024](https://carpentries.org/blog/2024/09/launching-the-carpentries-membership-program-survey/) and again in [September 2025](https://carpentries.org/blog/2025/09/share-your-input-on-the-carpentries-new-membership-model/), we are so pleased to announce that we have redesigned our Membership program into a new model called the **Partnership program** that can be tailored to suit varying institutional needs. Plans are underway to launch the redesigned program in April this year!

Reviewer: ```suggestion
With much-valued input from our community, we have been working over the past few years on ways to improve our Membership program to meet the needs of our community’s partner organisations and institutions. After considering and engaging with feedback from our community surveys from [September 2024](https://carpentries.org/blog/2024/09/launching-the-carpentries-membership-program-survey/) and again in [September 2025](https://carpentries.org/blog/2025/09/share-your-input-on-the-carpentries-new-membership-model/), we are so pleased to announce that we have redesigned our Membership program into a new model called the **Partnership program** that can be tailored to suit varying institutional needs. Plans are underway to launch the redesigned program in April this year!
```

Feedback from our community pointed out that our existing Membership tiers do not always align perfectly with all institutions’ needs. (For a summary of the feedback that was collated from the community surveys, please check out the feedback reports prepared for The Carpentries by the consulting agency, Wildly Open.) In response to this feedback, we have designed our new Partnership program to be more flexible and to offer a wider variety of benefit options. Next time your institution enters a Partnership agreement with The Carpentries or renews an existing Membership, they can now design a **customised Partnership package** that is targeted towards the institution’s specific needs. 

## Continuing and new offerings 

Existing Membership benefits, which will continue to be available through Partnership, include:

* Multi-day trainings leading to certification - including Instructor Training and Collaborative Lesson Development Training.
* Multi-day workshops in data science and computational skills (Data Carpentry, Library Carpentry, and Software Carpentry workshops). 

> In addition, the Partnership program will make new service offerings available, including shorter-format professional development training for faculty, researchers, staff, and students. New offerings will include:  

Reviewer: ```suggestion
In addition, the Partnership program will introduce new service offerings, including shorter-format professional development training for faculty, researchers, staff, and students. New offerings will include:  
```

## Monetary discounts for select community service roles

Furthermore, the Partnership program is designed to strengthen the two-way relationship between The Carpentries and our Partner organisations. To emphasise the bi-directionality of these relationships, Partners will be strongly encouraged to support community service from their staff back to the global Carpentries community. Community service roles that directly support The Carpentries’ long-term sustainability (including Instructor Trainers, Code of Conduct Committee members, and Lesson Developer Trainers) will be recognised through monetary discounts towards their organisation’s Partnership. 

Through these discounts, we want to more clearly demonstrate the value of our community members’ Carpentries services to institutional administrators and help shift the local incentivisation structure, while further strengthening the global Carpentries community.

## What is next? 

> If your organisation is already a Member, they will be invited to set up a Partnership at their next renewal. If your organisation does not yet have a Membership, we invite you to connect with us to help establish a Partnership at your institution. You can connect with us at [membership@carpentries.org](mailto:membership@carpentries.org) or [schedule a meeting](https://calendly.com/carpentries-membership/15-minutes-membership-questions-support). 

Reviewer: ```suggestion
If your organisation is already a Member, you will be invited to set up a Partnership at your next renewal. If your organisation does not yet have a Membership, we invite you to connect with us to help establish a Partnership at your institution. You can connect with us at [membership@carpentries.org](mailto:membership@carpentries.org) or [schedule a meeting](https://calendly.com/carpentries-membership/15-minutes-membership-questions-support). 
```

> We hope you’re as excited about these upcoming changes as we are. Please do not hesitate to reach out with any questions or concerns (or excitement!) between now and April. We are excited to engaging with you and your organisations as Partners in the coming year!

Reviewer: ```suggestion
We hope you’re as excited about these upcoming changes as we are. Please do not hesitate to reach out with any questions or concerns (or excitement!) between now and April. We are excited to engage with you and your organisations as Partners in the coming year!
```
